# 1. Contexto del problema

La primera pestaña del tablero presenta la pregunta de investigación, el planteamiento del
problema y los datos empleados, junto con cuatro indicadores que resumen la escala del
análisis. Este capítulo reproduce esos elementos y desarrolla el razonamiento que el tablero
condensa en pocos párrafos.

In [1]:
import json
import pandas as pd
import plotly.express as px

caudales = pd.read_csv("datos/caudales.csv", parse_dates=["fecha"])
ubicaciones = pd.read_csv("datos/ubicaciones.csv", sep=";", encoding="utf-8-sig")
par = json.load(open("datos/parametros.json", encoding="utf-8"))

Q_EF, LIM = par["Q_EF"], par["LIM"]

## 1.1 Indicadores generales

Las cuatro tarjetas que encabezan la pestaña muestran la descarga efectiva, la banda de
caudales que define la clase positiva, el periodo de registro y el número de observaciones
diarias. Sus valores se obtienen directamente de la serie de caudales y de los parámetros
estimados en el análisis exploratorio.

In [2]:
pd.DataFrame({
    "Indicador": ["Descarga efectiva", "Banda de transporte dominante",
                  "Periodo de registro", "Observaciones diarias"],
    "Valor": [f"{Q_EF:,.0f} m³/s",
              f"{LIM[0]:,.0f} – {LIM[1]:,.0f} m³/s",
              f"{caudales.fecha.min():%Y} – {caudales.fecha.max():%Y}",
              f"{len(caudales):,}"],
})

,Indicador,Valor
0,Descarga efectiva,"9,954 m³/s"
1,Banda de transporte dominante,"8,461 – 11,448 m³/s"
2,Periodo de registro,1940 – 2026
3,Observaciones diarias,"31,475"


La descarga efectiva y la banda de ±15 % que la rodea se derivan en el capítulo 2 a
partir del análisis magnitud-frecuencia de la estación Calamar. El periodo de registro, que
se extiende de 1940 a 2026, corresponde a esa misma estación, mientras que las estaciones del
bajo Magdalena inician su operación entre 1972 y 1975, de modo que el número de
observaciones diarias no equivale al número de días con información completa en toda la
red.

## 1.2 Pregunta de investigación

> ¿Es posible clasificar, con la información hidrológica disponible hasta el día $t$, si el
> día $t+15$ pertenecerá al rango de descarga efectiva del río Magdalena en la estación
> Calamar?

La pregunta define una tarea de clasificación binaria en la que cada día del registro
constituye una observación. Los predictores se construyen con el caudal observado hasta el
día $t$ en siete estaciones del cauce, y la etiqueta indica si el caudal medio del día $t+15$
en Calamar cae dentro de la banda de descarga efectiva. El horizonte de quince días responde
a un compromiso entre la utilidad operativa de la anticipación y la dificultad del problema,
cuya justificación empírica se presenta en el capítulo 3.

## 1.3 Planteamiento del problema

Un río transporta, además de agua, partículas de suelo y roca desprendidas de las laderas de
su cuenca, material que viaja en suspensión o rodando por el fondo del cauce. Ese transporte
determina en buena medida la forma del río, dado que el sedimento erosiona unos tramos y se
deposita en otros, y condiciona la vida útil de los embalses, la profundidad disponible para
la navegación y el crecimiento de los deltas.

El río Magdalena drena cerca de la cuarta parte del territorio colombiano, atraviesa una
cordillera joven y tectónicamente activa y recibe precipitaciones intensas, condiciones que
en conjunto producen uno de los rendimientos sedimentarios más altos del mundo por unidad de
área drenada (Milliman y Syvitski, 1992). A escala de subcuenca, la pendiente del terreno y
la escorrentía resultan los factores que mejor explican ese rendimiento (Restrepo et al.,
2006), y los cambios de uso del suelo habrían incrementado el aporte sedimentario respecto
de las condiciones previas a la intervención antrópica (Restrepo y Syvitski, 2006). El
registro instrumental muestra, además, una variabilidad interanual considerable asociada a
la alternancia de las fases del fenómeno ENSO (Restrepo y Kjerfve, 2000).

El transporte acumulado a lo largo de los años depende de dos factores que operan en sentidos
opuestos. La cantidad de sedimento que moviliza un caudal crece con su magnitud, mientras que
la frecuencia con que ese caudal se presenta disminuye a medida que aumenta. El producto de
ambos factores alcanza su máximo en un valor intermedio del rango, denominado descarga
efectiva, que se interpreta como el caudal responsable de la configuración geométrica del
cauce por ser el que acumula el mayor trabajo geomorfológico a largo plazo (Wolman y Miller,
1960). Su cálculo se sistematizó mediante el análisis magnitud-frecuencia (Andrews, 1980),
cuyo resultado depende del número de clases y de la curva de transporte empleada (Nash,
1994), y su equivalencia con el caudal a cauce lleno solo se cumple bajo determinadas
condiciones hidráulicas (Emmett y Wolman, 2001).

La descarga efectiva caracteriza el comportamiento sedimentario de largo plazo, aunque su
valor no indica en qué momento volverá a presentarse. La operación de embalses requiere
anticipar la llegada de sedimento para programar las purgas, y el mantenimiento del canal
navegable requiere prever los periodos de mayor depositación para planificar los dragados.
Por esa razón, este trabajo evalúa si el estado hidrológico registrado en un conjunto de
estaciones distribuidas sobre el cauce permite anticipar los días en que el río se situará en
el rango de transporte dominante. La hipótesis de trabajo sostiene que el caudal registrado
aguas arriba, combinado con el estado reciente de la estación objetivo y con la posición
dentro del ciclo anual, contiene señal suficiente para realizar esa anticipación con quince
días de antelación.

## 1.4 Datos y delimitación

Los datos corresponden a las series de caudal medio diario de siete estaciones operadas por
el IDEAM, descargadas del portal DHIME (IDEAM, 2026). La estación objetivo es Calamar,
situada en el ápice del delta, en el departamento de Bolívar, y las seis restantes se
distribuyen aguas arriba, desde Guaduas, en Cundinamarca, hasta la depresión momposina. El
estudio se limita al cauce principal del Magdalena, de manera que el aporte del Cauca y de los
demás afluentes se manifiesta de forma indirecta a través del caudal de las estaciones del
bajo Magdalena.

En el plano temporal, el modelo se ajusta con la información anterior al 1 de enero de 2010 y
se evalúa sobre el periodo posterior, que se extiende hasta septiembre de 2026. En el plano
temático, el trabajo anticipa el régimen de caudal asociado al transporte dominante y no la
cantidad de sedimento transportada, restricción que obedece a la naturaleza de la serie de
transporte publicada y que se documenta en el capítulo 2. La licencia del portal autoriza la
descarga para uso personal y no comercial, por lo que los archivos originales no se
redistribuyen; el repositorio incluye únicamente la serie diaria consolidada que emplean el
tablero y este informe.

El mapa del tablero ubica las siete estaciones sobre una capa de OpenStreetMap y las colorea
según su altitud. Al pasar el cursor sobre cada punto se despliegan el municipio, el
departamento y la altitud de la estación.

In [3]:
fig = px.scatter_map(
    ubicaciones, lat="Latitud", lon="Longitud",
    hover_name="Estacion",
    hover_data={"Municipio": True, "Departamento": True,
                "Altitud (m s. n. m.)": True, "Latitud": False, "Longitud": False},
    color="Altitud (m s. n. m.)", color_continuous_scale="Turbo",
    zoom=4.6, map_style="open-street-map", height=460)
fig.update_traces(marker=dict(size=14))
fig.update_layout(margin=dict(l=0, r=0, t=10, b=0))
fig

In [4]:
ubicaciones

,Estacion,Codigo IDEAM,Departamento,Municipio,Latitud,Longitud,Altitud (m s. n. m.)
0,ARRANCAPLUMAS - AUT,21237020,Cundinamarca,Guaduas,5.202417,-74.727611,222
1,PUERTO SALGAR - AUT,23037010,Cundinamarca,Puerto Salgar,5.469667,-74.662167,168
2,TRES CRUCES,25027640,Bolivar,Achí,8.703083,-74.517306,31
3,COYONGAL,25027930,Bolivar,Magangué,8.915130,-74.485540,20
4,EL BANCO - AUT,25027020,Magdalena,El Banco,8.992528,-73.969444,29
5,SAN ROQUE,25027320,Magdalena,El Banco,9.071622,-74.157176,24
6,CALAMAR,29037020,Bolivar,Calamar,10.244042,-74.914705,8


Las estaciones se ordenan a lo largo de un gradiente altitudinal que desciende de 222 m s.
n. m. en Arrancaplumas, en el municipio de Guaduas, a 8 m s. n. m. en Calamar. Las dos
estaciones del alto Magdalena, Arrancaplumas y Puerto Salgar, se encuentran a más de 160 m s.
n. m., mientras que las cuatro del bajo Magdalena se sitúan entre 20 y 31 m s. n. m. Ese orden
geográfico no interviene en la construcción del modelo, aunque se emplea en el capítulo 2 como
referencia externa para contrastar los tiempos de tránsito estimados a partir de las series de
caudal.